core/src/main/java/org/apache/accumulo/core/file/rfile/RFileOperations.java

```diff
@@ -129,7 +136,23 @@ protected FileSKVWriter openWriter(FileOptions options) throws IOException {
       String file = options.getFilename();
       FileSystem fs = options.getFileSystem();
 
-      outputStream = fs.create(new Path(file), false, bufferSize, (short) rep, block);
+      if (options.dropCacheBehind) {
+        EnumSet<CreateFlag> set = EnumSet.of(CreateFlag.SYNC_BLOCK, CreateFlag.CREATE);
+        outputStream = fs.create(new Path(file), FsPermission.getDefault(), set, bufferSize,
```

Comment:
`block` is passed to the other call of fs.create(), but not this one.  Is there a method that supports that and the perms?

Reply:
I'll look.

Reply:
turns out that `block` will work there, there is another variable called `blockSize` that was put there by mistake. Fixed in f5aaebb.